src/components/screens/SuggestedApps.tsx

```diff
@@ -84,7 +84,12 @@ const GridContainer = styled.div`
 export default function SuggestedApps() {
   return (
     <>
-      <Text variant="extraSmall" color="neutral.c70" marginBottom={6}>
+      <Text
+        fontSize={6}
```

Comment:
Changing this you need to also update the recently used title too

Reply:
<img width="1494" alt="Screenshot 2025-04-01 at 18 24 39" src="https://github.com/user-attachments/assets/26ea95fd-ab31-4aca-a9a1-9e4c0d091698" />